documentation/whatsnew/forge/forge-news-4.3.1.Final.adoc

```diff
@@ -0,0 +1,25 @@
+= Forge Tools 4.3.1.CR1 What's New
+:page-layout: whatsnew
+:page-component_id: forge
+:page-component_version: 4.3.1.Final
+:page-product_id: jbt_core
+:page-product_version: 4.3.1.Final
+
+ifndef::finalnn[]
```

Comment:
So you don't want to include forge runtime update into N&N at all? This is the final N&N.